tests/test_cache.py

```diff
@@ -15,3 +15,85 @@
 
 from unittest import TestCase
 from c7n import cache
+from argparse import Namespace
+import cPickle
+import tempfile
+import mock
+
+
+class FileCacheManagerTest(TestCase):
+    def setUp(self):
+        self.test_config = Namespace(cache_period=60, cache='test-cloud-custodian.cache')
+        self.test_cache = cache.FileCacheManager(self.test_config)
+        self.test_key = 'test'
+        self.bad_key = 'bad'
+        self.test_value = [1, 2, 3]
+
+
+    def test_get(self):
+        #mock the pick and set it to the data variable
+        test_pickle = cPickle.dumps({cPickle.dumps(self.test_key): self.test_value}, protocol=2)
+        self.test_cache.data = cPickle.loads(test_pickle)
+
+        #assert
+        self.assertEquals(self.test_cache.get(self.test_key), self.test_value)
+        self.assertEquals(self.test_cache.get(self.bad_key), None)
+
+
+
+    def test_load(self):
```

Comment:
this test can be nuked or implemented, but i'd rather not empty skeletons if only because it artificially inflates the test case count.